src/test/kotlin/uk/gov/justice/digital/hmpps/hmppstier/integration/bdd/BddSteps.kt

```diff
@@ -101,6 +101,70 @@ class BddSteps : En {
     Given("an offender is {string}") { gender: String ->
       setupData.setGender(gender)
     }
+
+    Given("an offender scores 21 points") {
+      setupData.setValidAssessment()
+      setupData.setOgrs("90") // 9 points
+      setupData.setNeeds(
+        mapOf(
+          "ACCOMMODATION" to "SEVERE",
+          "EDUCATION_TRAINING_AND_EMPLOYABILITY" to "SEVERE",
+          "RELATIONSHIPS" to "SEVERE",
+          "LIFESTYLE_AND_ASSOCIATES" to "SEVERE",
+          "DRUG_MISUSE" to "SEVERE",
+          "ALCOHOL_MISUSE" to "SEVERE"
+        )
+      ) // 12 points
+    }
+
+    Given("an offender scores 20 points") {
+      setupData.setValidAssessment()
+      setupData.setOgrs("100") // 10 points
+      setupData.setNeeds(
+        mapOf(
+          "ACCOMMODATION" to "SEVERE",
```

Comment:
Very minor but it would be nice to use the full range of needs across these tests